gramps/plugins/lib/libmetadata.py

```diff
@@ -97,159 +112,284 @@ def format_gps(raw_dms, nsew):
-        (DESCRIPTION, 'Exif.Image.Rating', None, None),
-        (IMAGE, 'Exif.Image.DocumentName', None, None),
-        (IMAGE, 'Exif.Photo.PixelXDimension', None, None),
-        (IMAGE, 'Exif.Photo.PixelYDimension', None, None),
-        (IMAGE, 'Exif.Image.XResolution', 'Exif.Image.ResolutionUnit', None),
-        (IMAGE, 'Exif.Image.YResolution', 'Exif.Image.ResolutionUnit', None),
-        (IMAGE, 'Exif.Image.Orientation', None, None),
-        (IMAGE, 'Exif.Photo.ColorSpace', None, None),
-        (IMAGE, 'Exif.Image.YCbCrPositioning', None, None),
-        (IMAGE, 'Exif.Photo.ComponentsConfiguration', None, None),
-        (IMAGE, 'Exif.Image.Compression', None, None),
-        (IMAGE, 'Exif.Photo.CompressedBitsPerPixel', None, None),
-        (IMAGE, 'Exif.Image.PhotometricInterpretation', None, None),
-        (CAMERA, 'Exif.Image.Make', None, None),
-        (CAMERA, 'Exif.Image.Model', None, None),
-        (CAMERA, 'Exif.Photo.FNumber', None, None),
-        (CAMERA, 'Exif.Photo.ExposureTime', None, None),
-        (CAMERA, 'Exif.Photo.ISOSpeedRatings', None, None),
-        (CAMERA, 'Exif.Photo.FocalLength', None, None),
-        (CAMERA, 'Exif.Photo.FocalLengthIn35mmFilm', None, None),
-        (CAMERA, 'Exif.Photo.MaxApertureValue', None, None),
-        (CAMERA, 'Exif.Photo.MeteringMode', None, None),
-        (CAMERA, 'Exif.Photo.ExposureProgram', None, None),
-        (CAMERA, 'Exif.Photo.ExposureBiasValue', None, None),
-        (CAMERA, 'Exif.Photo.Flash', None, None),
-        (CAMERA, 'Exif.Image.FlashEnergy', None, None),
-        (CAMERA, 'Exif.Image.SelfTimerMode', None, None),
-        (CAMERA, 'Exif.Image.SubjectDistance', None, None),
-        (CAMERA, 'Exif.Photo.Contrast', None, None),
-        (CAMERA, 'Exif.Photo.LightSource', None, None),
-        (CAMERA, 'Exif.Photo.Saturation', None, None),
-        (CAMERA, 'Exif.Photo.Sharpness', None, None),
-        (CAMERA, 'Exif.Photo.WhiteBalance', None, None),
-        (CAMERA, 'Exif.Photo.DigitalZoomRatio', None, None),
-        (GPS, 'Exif.GPSInfo.GPSLatitude',
-              'Exif.GPSInfo.GPSLatitudeRef', format_gps),
-        (GPS, 'Exif.GPSInfo.GPSLongitude',
-              'Exif.GPSInfo.GPSLongitudeRef', format_gps),
-        (GPS, 'Exif.GPSInfo.GPSAltitude',
-              'Exif.GPSInfo.GPSAltitudeRef', None),
-        (GPS, 'Exif.GPSInfo.GPSTimeStamp', None, None),
-        (GPS, 'Exif.GPSInfo.GPSSatellites', None, None),
-        (ADVANCED, 'Exif.Image.Software', None, None),
-        (ADVANCED, 'Exif.Photo.ImageUniqueID', None, None),
-        (ADVANCED, 'Exif.Image.CameraSerialNumber', None, None),
-        (ADVANCED, 'Exif.Photo.ExifVersion', None, None),
-        (ADVANCED, 'Exif.Photo.FlashpixVersion', None, None),
-        (ADVANCED, 'Exif.Image.ExifTag', None, None),
-        (ADVANCED, 'Exif.Image.GPSTag', None, None),
-        (ADVANCED, 'Exif.Image.BatteryLevel', None, None)]
+def get_NamedRegions(self, key, metadata, image_path, mypixbuf):
```

Comment:
This should be lowercase.  Check the [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names) naming conventions.

Reply:
Done